src/rpc/client/ts/src/server.ts

```diff
@@ -80,22 +51,24 @@ export async function startServer(
     rootPath: rootPath,
     port: port,
   })
-  const artifact = new Artifact('omega-edit-grpc-server', version, rootPath)
 
+  const scriptDir = path.join(rootPath, 'node_modules/omega-edit/bin')
   let args = [`--interface=${host}`, `--port=${port}`]
```

Comment:
This should be const, and it should also include `--pidfile=${pidfile}` using the new pidfile option to the server.